cms/admin/forms.py

```diff
@@ -455,9 +452,7 @@ def get_or_create_root(self):
         Creates the root node used to store all page types
         for the current site if it doesn't exist.
         """
-        root_page = PageType.get_root_page(site=self._site)
-
-        if not root_page:
+        if not (root_page := PageType.get_root_page(site=self._site)):
```

Comment:
I would like to opt for better readability here and avoid the walrus operator at least in a nested statement (if not at all).

```suggestion
        root_page = PageType.get_root_page(site=self._site)
        if not root_page:
```